Aliases/qt6

```diff
@@ -0,0 +1 @@
+../Formula/qt@6.rb
```

Comment:
missing an `Aliases/qt` for people to use `qt` as a dependency

Reply:
Form Qt community should keep qt5 and qt6 at same time, like python2 and python3. Some formulae can not be built under qt6.

Reply:
Then they aren't following the homebrew naming conventions. QT MUST point to the latest version of QT.

Reply:
Ok, then the qt.rb must be qt@5.rb?